cli/remove/remove.go

```diff
@@ -0,0 +1,71 @@
+package remove
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// Remove removes binary/directory and symlinks.
+func Remove(programm string, dst string) error {
+	if dst == "" {
+		dst, _ = os.Getwd()
+		dst += "/bin"
+	}
+	path := dst + "/" + programm
+
+	var linkPath string
+	var err error
+	if strings.Contains(programm, "tt") {
+		linkPath, err = filepath.Abs(dst + "/tt")
+		if err != nil {
+			return err
+		}
+	} else if strings.Contains(programm, "tarantool") {
+		linkPath, _ = filepath.Abs(dst + "/tarantool")
+	} else {
+		return fmt.Errorf("Unknown programm: %s", programm)
+	}
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return fmt.Errorf("There is no %s installed.", programm)
+	}
+	log.Warnf("%s found, removing...", programm)
```

Comment:
Why is program removing message in remove command a warning?

Reply:
changed